tests/queries/0_stateless/02554_log_faminy_support_storage_policy.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS test_2554_log;
+CREATE TABLE test_2554_log (n UInt32) ENGINE = Log SETTINGS storage_policy = 'default';
+
+INSERT INTO test_2554_log SELECT 1;
+SELECT * FROM test_2554_log;
+
+DROP TABLE test_2554_log;
+
+DROP TABLE IF EXISTS test_2554_tinylog;
+CREATE TABLE test_2554_tinylog (n UInt32) ENGINE = Log SETTINGS storage_policy = 'default';
+
+INSERT INTO test_2554_tinylog SELECT 1;
+SELECT * FROM test_2554_tinylog;
+
+DROP TABLE test_2554_tinylog;
+
+DROP TABLE IF EXISTS test_2554_stripelog;
+CREATE TABLE test_2554_stripelog (n UInt32) ENGINE = StripeLog SETTINGS storage_policy = 'default';
```

Comment:
it could be better to make a test with non default policy, because we cannot verify that the setting is used, because if it is not specified, the policy will anyway be `default`.

Reply:
Which storage_policy can I specify in the test?

Reply:
any from `tests/configs/config.d/storage_conf.xml`